kubetest/e2e.go

```diff
@@ -615,33 +620,52 @@ func kubemarkUp(dump string, o options, deploy deployer) error {
 		return err
 	}
 
-	masterIP, err := control.Output(exec.Command(
-		"gcloud", "compute", "addresses", "describe",
-		os.Getenv("MASTER_NAME")+"-ip",
-		"--project="+o.gcpProject,
-		"--region="+o.gcpZone[:len(o.gcpZone)-2],
-		"--format=value(address)"))
-	if err != nil {
-		return fmt.Errorf("failed to get masterIP: %v", err)
+	var masterIP, masterInternalIP []byte
+
+	if o.deployment == "bash" && o.provider == "gce" {
```

Comment:
n.b.

Reply:
any specical reasons to check `o.deployment == "bash"`?

Reply:
https://github.com/kubernetes/test-infra/pull/16452#discussion_r390901322

Reply:
thanks